charts/rancher-monitoring/v0.0.3/charts/prometheus/templates/rbac.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and .Values.enabledRBAC (not .Values.serviceAccountName) }}
+{{- if and .Values.enabledRBAC (not .Values.global.rancherInjectedServiceAccountName) }}
```

Comment:
above, you added to this if check like:
```
{{- if and .Values.enabledRBAC (not .Values.serviceAccountName) (not .Values.global.rancherInjectedServiceAccountName) }}
```
how do you decide when to add an additional check vs replace the check